.vscode/extensions.json

```diff
@@ -3,7 +3,15 @@
   // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
 
   // List of extensions which should be recommended for users of this workspace.
-  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode"],
+  "recommendations": [
+    "bradlc.vscode-tailwindcss",
+    "dbaeumer.vscode-eslint",
+    "JuanBlanco.solidity",
+    "redhat.vscode-yaml",
+    "rvest.vs-code-prettier-eslint",
+    "tintinweb.solidity-visual-auditor",
+    "tintinweb.vscode-solidity-language"
```

Comment:
What does this one provide? Does the syntax highlighting conflict with `JuanBlanco.solidity`?

Reply:
@holic I don't think it does but I will double check this with a clean VS Code install. I'll also include some documentation/links about what all the extensions do and other recommended VS Code configuration.